src/models/ChannelMember.js

```diff
@@ -5,7 +5,7 @@ class ChannelMember extends Base {
     super(client);
 
     this.id = data?.id;
-    this.targetChannelId = data?.targetGroupId;
```

Comment:
this.targetChannelId = data?.targetGroupId;
  This line is still correct

Reply:
No Channel member already have data from framework with Channel Id 

Data object :
```js
{
  id: 38025651,
  capabilities: 1,
  hash: 'vv2bcWBx',
  targetChannelId: 18404568
}

```